model/CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 Changelog for `twilight-model`.
 
+## [0.4.3] - 2021-06-12
+
+### Additions
+
+Support `StageInstanceCreate`, `StageInstanceDelete`, `StageInstanceUpdate` events ([#845] -
+[@7596ff]).
+
+Support `channel::stage_instance::StageInstance::{discoverable_disabled, privacy_level}` and
+`channel::stage_instance::PrivacyLevel` ([#867] - [@7596ff]).
+
+Support `CurrentApplicationInfo::{flags, privacy_policy_url, terms_of_service_url}` ([#882], [#904]
+- [@7596ff], [@Gelbpunkt])
+
+Support `StageInstance*` events and `PrivacyLevel` change key in the audit log ([#907] - [@7596ff]).
+
+Support `IntegrationCreate`, `IntegrationDelete`, and `IntegrationUpdate` events ([#914] -
+[@7596ff]).
+
+### Changes
+
+Support `Guild::stage_instances`, which are present in the `GuildCreate` event ([#845] - [@7596ff]).
+
+Deprecate `{Guild, PartialGuild}::nsfw` in favor of `nsfw_level`, and add its change key in audit
```

Comment:
This should be split into a mention of a deprecation and a mention in the Additions section, both referencing the same PR